python/amici/sbml_import.py

```diff
@@ -76,58 +76,59 @@ class SbmlImporter:
 
     """
 
-    def __init__(self, SBMLFile, show_sbml_warnings=False):
-        """Create a new Model instance.
+    def __init__(
```

Comment:
can we please the doxygen docstring formating. I know this sucks but doxygen is the only thing that works reliably across multiple languages and none of the python filter that I know support sphinx style documentation (yes I know this sucks but theres not much we can do without investing a ridiculous amount of time)

Reply:
yes, totally makes sense. hope it's doxygen compatible now.